packages/adp-tooling/test/unit/writer/index.test.ts

```diff
@@ -40,7 +40,7 @@ describe('ADP writer', () => {
     describe('generate', () => {
         test('minimal config', async () => {
             const projectDir = join(outputDir, 'minimal');
-            await generate(projectDir, config, fs);
+            await generate(projectDir, config);
```

Comment:
What is the reason for removing the parameter `fs` here?

Reply:
Good point but we use the `fs` instance here to be able to write the output to the filessystem for debugging (check the `afterAll` function), thereofre, please revert your change here

Reply:
done